packages/react-components/react-carousel-preview/src/components/CarouselNavImageButton/CarouselNavImageButton.types.ts

```diff
@@ -1,7 +1,15 @@
+import { ARIAButtonSlotProps } from '@fluentui/react-aria';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type CarouselNavImageButtonSlots = {
-  root: Slot<'div'>;
+  /**
+   * ARIA compliant nav buttons used to jump to pages
+   */
+  root: NonNullable<Slot<ARIAButtonSlotProps<'a'>>>;
```

Comment:
Are anchor tags required for carousel nav buttons? All interactions should be covered by buttons, and I don't forsee any scenarios where the nav buttons would actually be links with href.

If you use a link without a href, it actually loses all link semantics. I think it would be better to keep it simple and stick to button

Reply:
Good catch, this was mostly just type safety so I assume we can use ARIAButtonSlotProps without the 'a' type specification? I'm not 100% familiar with a regular button slot vs ARIAButtonSlotProps